package.json

```diff
@@ -86,9 +86,12 @@
     "chai": "^4.3.0",
     "chai-as-promised": "^7.1.1",
     "ember-cli": "~4.4.0",
+    "ember-cli-addon-docs": "^4.2.2",
+    "ember-cli-addon-docs-yuidoc": "^1.0.0",
     "ember-cli-deploy": "^1.0.2",
     "ember-cli-deploy-build": "^2.0.0",
     "ember-cli-deploy-git": "^1.3.4",
+    "ember-cli-deploy-git-ci": "^1.0.1",
```

Comment:
To set up deployment (last commit, commit 35), I ran `ember generate ember-cli-addon-docs` following the [documentation](https://ember-learn.github.io/ember-cli-addon-docs/docs/deploying).

The command ended up installing `ember-cli-deploy-git-ci`, but I am unsure if we need it. The command would have also installed the following, had they not been installed already:

- `ember-cli-deploy`
- `ember-cli-deploy-build`
- `ember-cli-deploy-git`

Reply:
Update: Do NOT install `ember-cli-deploy-git-ci`. See https://github.com/ember-intl/ember-intl/pull/1692.

Reply:
Update: Do NOT install `ember-cli-deploy-git-ci`. See https://github.com/ember-intl/ember-intl/pull/1692.